sql-parser/src/main/java/io/crate/sql/ExpressionFormatter.java

```diff
@@ -314,9 +314,17 @@ protected String visitFunctionCall(FunctionCall node, @Nullable List<Expression>
                 builder.append('(').append(arguments).append(')');
             }
 
-            if (node.getWindow().isPresent()) {
-                builder.append(" OVER ").append(visitWindow(node.getWindow().get(), parameters));
-            }
+            node.filter()
+                .map(filter -> builder
```

Comment:
I think using `ifPresent` instead of `map` would make more sense here since the reslt of the mapping function is not used.